Base/res/ladybird/inspector.css

```diff
@@ -204,7 +218,7 @@ details > :not(:first-child) {
 }
 
 .console {
-    font-family: Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+    font-family: var(--code-font-family, monospace);
```

Comment:
Do we still need `, monospace` if that's in the var?

Reply:
It's a bit belt-and-braces; the `monospace` is a fallback in case the var doesn't exist. Probably best to remove that actually, so we know if we remove the var.